### ISSUE TITLE
tinychat.serve.model_worker_new.py AWQ model in training mode

### ISSUE THREAD

**Issue by NigelNelson** [mit-han-lab/llm-awq#186]
When running, I get the following output:
```bash
2024-05-14 14:10:44 | WARNING | root | Caution: Your LLM is currently in training mode, ensuring accurate gradient computation. Please be vigilant, particularly regarding BatchNorm and Dropout operations.
2024-05-14 14:10:44 | WARNING | root | Caution: Your LLM is currently in training mode, ensuring accurate gradient computation. Please be vigilant, particularly regarding BatchNorm and Dropout operations.
2024-05-14 14:10:44 | WARNING | root | Caution: Your LLM is currently in training mode, ensuring accurate gradient computation. Please be vigilant, particularly regarding BatchNorm and Dropout operations.
2024-05-14 14:10:44 | WARNING | root | Caution: Your LLM is currently in training mode, ensuring accurate gradient computation. Please be vigilant, particularly regarding BatchNorm and Dropout operations.
2024-05-14 14:10:44 | WARNING | root | Caution: Your LLM is currently in training mode, ensuring accurate gradient computation. Please be vigilant, particularly regarding BatchNorm and Dropout operations.
2024-05-14 14:10:44 | WARNING | root | Caution: Your LLM is currently in training mode, ensuring accurate gradient computation. Please be vigilant, particularly regarding BatchNorm and Dropout operations.
2024-05-14 14:10:44 | WARNING | root | Caution: Your LLM is currently in training mode, ensuring accurate gradient computation. Please be vigilant, particularly regarding BatchNorm and Dropout operations.
2024-05-14 14:10:44 | WARNING | root | Caution: Your LLM is currently in training mode, ensuring accurate gradient computation. Please be vigilant, particularly regarding BatchNorm and Dropout operations.
2024-05-14 14:10:44 | WARNING | root | Caution: Your LLM is currently in training mode, ensuring accurate gradient computation. Please be vigilant, particularly regarding BatchNorm and Dropout operations.
2024-05-14 14:10:44 | WARNING | root | Caution: Your LLM is currently in training mode, ensuring accurate gradient computation. Please be vigilant, particularly regarding BatchNorm and Dropout operations.
...
```
Looks like model.eval() needs to be called somewhere...
